recipes/tf_object_detection/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "tf_object_detection" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/Maxar-Corp/ratpac/releases/download/od-{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 476eaf754c61c5afce39e36683bb4a7a6b7f9084e2bc89f075d58e294da27768
+
+build:
+  number: 0
+  skip: true  # [win]  # Skip all windows for now due to numpy/scipy version pinning issues
+  skip: True  # [win32]
+  skip: True  # [win64 and py<35]
+  skip: True  # [osx and py34]
+  skip: True  # [py26]
+  skip: True  # [py33]
```

Comment:
We don't build on python 2.6, 3.3, 3.4, or 3.5.  So these skips are not needed

Reply:
thanks. I used the `tf-feedstock` recipe as an example which have these skips. I'will remove them.